### PR TITLE
Added travis ci support 

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

**Comment by kyldvs** [comment]
I have no experience with travis ci, @zpao does this seem okay?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+sudo: false
+
+language: node_js
+
+node_js:
+  - "4"
+  - "5"
+  - "6"
```
Comment (zpao) [inline .travis.yml]: Let's just do 4 and 6. We get next to nothing from testing 5 except longer build times.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+script:
+  - npm run build
+  - npm run test
+
+# Necessary to compile native modules for io.js v3 or Node.js v4
```
Comment (zpao) [inline .travis.yml]: We don't need to mention io.js :)

**Comment by zpao** [comment]
Just my comments and this seems fine. We can always tweak it later.

**Comment by amilajack** [comment]
@zpao great! Sounds good.

**Comment by amilajack** [comment]
Fixed

**Comment by alexeyraspopov** [comment]
Updated PR's title is confusing :)

**Comment by kyldvs** [comment]
Thanks for setting this up @amilajack!

**Comment by amilajack** [comment]
@zpao @kyldvs also plz remember to enable the tests a [travis-ci.org](http://travis-ci.org) :)

**Comment by zpao** [comment]
@amilajack Organization privileges means somebody else has to do it, but we've asked. It'll be on soon :)

**Pull request merged**